test/e2e/node/node_problem_detector.go

```diff
@@ -131,44 +131,20 @@ var _ = SIGDescribe("NodeProblemDetector", func() {
 				framework.ExpectEqual(result.Code, 0)
 				gomega.Expect(result.Stdout).NotTo(gomega.ContainSubstring("node-problem-detector.service: Failed"))
 
-				// Let's assume that node problem detector has started the same time as kubelet
-				// We only will check for the KubeletStart if parsing of date here succeed
-				// This is an optimization to not ssh one more time to get kubelet's start time.
-				// Also we assume specific datetime format to simplify the logic
-				output := result.Stdout
-
-				// searching for the line "Apr 14 04:47:42 gke-cluster-1-default-pool-b1565719-eqht systemd[1]: Started Kubernetes node problem detector." and fallback to
-				idx := strings.Index(output, "Started Kubernetes node problem detector")
-				if idx != -1 {
-					output = output[:idx]
-					idx = strings.LastIndex(output, "\n")
-
-					if idx != -1 {
-						output = output[0:15]
-					}
-
-					st, err := time.Parse("Jan 02 15:04:05", output)
-					st = st.AddDate(time.Now().Year(), 0, 0)
+				// We only will check for the KubeletStart even if parsing of date here succeeded.
+				ginkgo.By(fmt.Sprintf("Check when node-problem-detector started on node %q", host))
+				npdStartTimeCommand := "sudo systemctl show --timestamp=utc node-problem-detector -P ActiveEnterTimestamp"
```

Comment:
I wanted to avoid additional command - see the comment. But it was indeed too fragile. Thank you for fixing